### DISCUSSION TITLE
Curso Fullstack .NET - Deploy

### DISCUSSION THREAD

**Discussion by aps180180** [balta-io/forum#657] [Tira Dúvidas] [answered]
Olá meu querido @andrebaltieri ! Gostaria de lhe fazer um pedido! Seria possível fazer uma aula fazendo deploy da aplicação na Azure?
Atenciosamente.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @aps180180 como vai?
Claro, já está no pipe!! Sexta sai um módulo novo, depois vamos pra PWA e depois o Deploy!

**Reply by aps180180** [reply]
Muito obrigado!
